apache/superset
# fix(datasets): give possibility to add dataset with slashes in name

### SUMMARY
Give possibility to users to add datasets which contains slashes in a table name


### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
#### Before

https://github.com/apache/superset/assets/66589759/dcf616e9-6775-4e88-8d7e-22d7e06cb00b


#### After

https://github.com/apache/superset/assets/66589759/2e850696-45ab-40e8-b74e-ede9726ab685


### TESTING INSTRUCTIONS
1. Go to "*/dataset/add/" URL, or push "+DATASET" button.
2. Choose database and schema.
3. Choose a table with "/" symbol in name
4. See error

### ADDITIONAL INFORMATION


- [x] Has associated issue: [23540](https://github.com/apache/superset/issues/23540)
